### ISSUE TITLE
crates.io release with fixed versions

### ISSUE THREAD

**Issue by SealedSun** [mitsuhiko/redis-rs#85]
The newly published `url` crate v1.0.0 comes with breaking changes. As of right now, the latest version of the `redis` crate on crates.io still has wildcard dependencies. On fresh projects or after a `cargo update`, you'll get errors like this

```
redis-0.5.2/src/connection.rs:138:17: 138:23 error: attempted to take value of method `scheme` on type `url::Url`
redis-0.5.2/src/connection.rs:138         if self.scheme == "redis" {
```

Sure, it's easy enough to work around this by
 * instead depending on the GitHub version of the `redis` crate
 * fixing the Cargo.lock file by hand (`url 0.5.9` worked for me)

but I think it would be worthwhile to have a version of the `redis` crate with fixed dependencies on crates.io.

**Comment by Manicqin** [comment]
Hi, The version you download from Crates.io is still before the fix. I guess it's because you didn't increment the version after fixing the dependencies. Thanks.

**Comment by lianhanloh** [comment]
Hi, running `cargo build` compiles with url v0.5.7 works. However, running `cargo package` tries to build with url 1.0.0 which breaks in the way described above.

**Comment by badboy** [comment]
I just published v0.5.3 with fixed dependency versions.
